### DISCUSSION TITLE
Remember Mission Selector Override

### DISCUSSION THREAD

**Discussion by Holgerhollebolle** [Foxlider/FASTER#154] [Ideas]
Hello, I want to use allways the same Override for Missions but switch the Mission it self. When I enable Mission Selector Override and input my Code the code is lost when I turn Override off and on again. I would like to have a way that Faster remembers my edits so I can use the same code for all Missions I start.
![image](https://github.com/Foxlider/FASTER/assets/10847197/ed0715a2-9130-41d4-98b5-3b0189399fc9)

**Comment by Holgerhollebolle** [comment]
Or is there another way to do this I miss?

**Reply by jupster** [reply]
The only way around it at the moment is to use Update-1.9

**Reply by Holgerhollebolle** [reply]
I am confused. I have Version 1.9d and it does not remember the Code in Mission selector Override. Could you explain how I can achieve that?

**Reply by jupster** [reply]
https://github.com/Foxlider/FASTER/tree/feature/Update-1.9 which has https://github.com/Foxlider/FASTER/pull/149 added to it.

**Reply by jupster** [reply]
Also this is in the works https://github.com/Foxlider/FASTER/issues/38 which is the exact thing your wanting.

**Reply by Holgerhollebolle** [reply]
@jupster Great.. Thanks. I will wait for the #38 Integration than. This is exactly what I want.
Btw...:  I use "LogObjectNotFound" parameter to prevent log Spaming of the Objectnotfound messages. But at a point the Headlessclients still spam the log with these messages...It would be totally great if you find a possibility to get rid of all these Messages and integrate a one button Option to turn these off. This is an issue soo many people complain about since many years.

**Reply by jupster** [reply]
No worry's. I'll look into the hc spamming and see how that goes

**Reply by Holgerhollebolle** [reply]
This would be great. if you can solve these Spam messages it would be a game changer. LogObjectNotFound is already helping a lot..but not with Headlessclients it seems.

**Comment by Holgerhollebolle** [comment]
I am talking about these Messages:

```
22:25:32 Client: Object 4:6110 (type Type_132) not found.
22:25:32 Client: Object 6:159 (type Type_132) not found.
22:25:32 Client: Object 4:6233 (type Type_131) not found.
22:25:33 Error: Object(4 : 6052) not found
22:25:33 Client: Object 4:6052 (type Type_125) not found.
22:25:33 Client: Object 6:13 (type Type_132) not found.
22:25:34 Client: Object 6:147 (type Type_165) not found.
22:25:34 Client: Object 6:13 (type Type_165) not found.
```
